zuul-core/src/test/java/com/netflix/zuul/netty/insights/ServerStateHandlerTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ *      Licensed under the Apache License, Version 2.0 (the "License");
+ *      you may not use this file except in compliance with the License.
+ *      You may obtain a copy of the License at
+ *
+ *          http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *      Unless required by applicable law or agreed to in writing, software
+ *      distributed under the License is distributed on an "AS IS" BASIS,
+ *      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *      See the License for the specific language governing permissions and
+ *      limitations under the License.
+ */
+
+package com.netflix.zuul.netty.insights;
+
+import static org.junit.Assert.assertEquals;
+import com.netflix.spectator.api.Counter;
+import com.netflix.spectator.api.DefaultRegistry;
+import com.netflix.spectator.api.Gauge;
+import com.netflix.spectator.api.Id;
+import com.netflix.spectator.api.Registry;
+import com.netflix.zuul.netty.insights.ServerStateHandler.InboundHandler;
+import com.netflix.zuul.netty.server.http2.DummyChannelHandler;
+import com.netflix.zuul.passport.CurrentPassport;
+import com.netflix.zuul.passport.PassportState;
+import io.netty.channel.embedded.EmbeddedChannel;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class ServerStateHandlerTest {
+
+    private Registry registry = new DefaultRegistry();
```

Comment:
would it make sense to move 'new DefaultRegistry()' to the init() method? 
(The init method is annotated with JUnit @Before)

Reply:
Yep, that's a miss. Fixed.